content/blog/a-guide-to-enable-managed-istio-service-mesh-in-a-kubernetes-cluster-on-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,214 @@
+                          service=productpage
+Annotations:              hpecp-internal-gateway/9080: epicgw.customer.hpe.net:10072
+Selector:                 app=productpage
+Type:                     NodePort
+IP Family Policy:         SingleStack
+IP Families:              IPv4
+IP:                       10.104.123.90
+IPs:                      10.104.123.90
+Port:                     http  9080/TCP
+TargetPort:               9080/TCP
+NodePort:                 http  31766/TCP
+Endpoints:                10.192.3.181:9080
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:
+  Type    Reason  Age   From         Message
+  ----    ------  ----  ----         -------
+  Normal  HpeCp   21s   hpecp-agent  Created HPECP K8S service
+```
+
+**Confirm the application is accessible from outside the cluster**
+
+Bookinfo application **productpage** can be accessed in the browser by typing the URL **http://epicgw.customer.hpe.net:10072**
+
+Note : Above URL might be different in your environment. You can form the URL by referring annotations from the **productpage** service.
+
+![](/img/bookinfo-productpage.png)
+
+![](/img/bookinfo-productpage-normal-user.png)
+
+**Monitor the sample application using the Kiali dashboard**
+
+Enter **bookinfo** into the field Filter by Namespace. The Kiali Overview screen displays the details about the namespace bookinfo. It shows that 4 applications are running in the **namespace bookinfo** with no inbound traffic.
+
+![](/img/kiali-bookinfo.png)
+
+In the **Graph** tab from the left navigation menu, after selecting the **namespace bookinfo**, the screen shows an overview topology of the Bookinfo application.
+
+![](/img/kiali-console-graph.png)
+
+In the **Applications** tab from the left navigation menu, after selecting the **namespace bookinfo**, the screen shows application details of the Bookinfo application.
+
+![](/img/kiali-console-applications.png)
+
+In the **Workloads** tab from the left navigation menu, after selecting the **namespace bookinfo**, the screen shows deployment details of the Bookinfo   application.
+
+![](/img/kiali-console-workloads.png)
+
+In the **Services** tab from the left navigation menu, after selecting the **namespace bookinfo**, you can check all the services details of the Bookinfo application.
+
+![](/img/kiali-console-services.png)
+
+**HPE GreenLake for Private Cloud Enterprise: Containers: Demo Summary**
+
+You can find the GitHub repository that hosts demo code **[here](https://github.com/cxteamtrials/caas-trials-content)**.
+
+We hope that this blog post has provided you with enough information for you to get started deploying containerized application or a managed service on a Kubernetes-based container stack using the cluster add-on feature provided by **HPE GreenLake for Private Cloud Enterprise: Containers**, refer to the **[HPE Developer Community blog](https://developer.hpe.com/)**.
```

Comment:
We hope that .....on a Kubernetes-based container stack using the Istio service mesh add-on provided by **HPE GreenLake for Private Cloud Enterprise: Containers**. 

To view more articles and tutorials on the use of the HPE GreenLake for Private Cloud Enterprise, refer to the **[HPE Developer Community Blog](https://developer.hpe.com)**